tests/ci/.mypy.ini

```diff
@@ -0,0 +1,16 @@
+[mypy]
+warn_no_return = False
+warn_unused_configs = True
+disallow_subclassing_any = True
+disallow_untyped_calls = False
```

Comment:
I did not check all options in this file, but at least `disallow_untyped_calls` is `False` by default already, i.e. can be removed (--> https://mypy.readthedocs.io/en/stable/config_file.html). Maybe take a look if there are more redundant options?

Reply:
Here's everything included in `strict` mode https://mypy.readthedocs.io/en/stable/config_file.html#confval-strict, but with explicit values. Explicit better than implicit, so I wouldn't change it.

Reply:
Besides that, defaults can change from version to version